src/main_thread/init/multi_thread_content_initializer.ts

```diff
@@ -91,6 +92,14 @@ export default class MultiThreadContentInitializer extends ContentInitializer {
    */
   private _currentMediaSourceCanceller: TaskCanceller;
 
+  /**
+   * Stores the resolvers and the current messageId that is sent to the web worker to receive segment sink metrics.
+   * The purpose of collecting metrics is for monitoring and debugging.
+   */
+  private _segmentMetrics: {
+    messageId: number;
```

Comment:
Could be clearer as `lastMessageId` or `nextMessageId` depending on which one it is